recipe/meta.yaml

```diff
@@ -31,17 +26,16 @@ requirements:
     - pkg-config
     - gperf
     - expat
-    - gettext                     # [not win]
+    - gettext     # [not win]
```

Comment:
We are skipping Windows, so we shouldn't need the selector

Reply:
Removed the selector